source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,107 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v7.9 - Feature Release](#release-v7-9-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 - [Release v7.7 - Feature Release](#release-v7-7-feature-release)
-- [Release v7.5 - Feature Release](#release-v7-5-feature-release)
-- [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.9 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: March 16th, 2023**
+
+### Important Upgrade Notes
+
+ - Added a new index on ``Posts(OriginalId)``. For a database with 11.8 million posts, on a machine with a i7-11800H CPU (8 cores, 16 threads), 32GiB of RAM and SSD, the index creation takes 98.51s on MYSQL and 2.6s on PostgreSQL. To avoid any table locking, servers using PostgreSQL are recommended to create the index online before performing the upgrade using the ``CONCURRENTLY`` option, as in ``CREATE INDEX CONCURRENTLY idx_posts_original_id ON Posts(OriginalId);``.
+
```

Comment:
- Disable global header. Admins can set the feature flag ``MM_FEATUREFLAGS_GLOBALDRAFTS`` to ``false`` to disable server-wide. It can't be disabled on a per-user basis. The code is located [here](https://github.com/mattermost/mattermost-server/blob/master/model/feature_flags.go#L11).
- Disable insights. Admins can set the feature flag ``MM_FEATUREFLAGS_INSGIHTSENABLED`` to ``false`` to disable server-wide. It can't be disabled on a per-user basis. The code is located [here](https://github.com/mattermost/mattermost-server/blob/master/model/feature_flags.go#L11).